pkgs/development/compilers/souffle/default.nix

```diff
@@ -19,6 +19,17 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-wdTBSmyA2I+gaSV577NNKA2oY2fdVTGmvV7h15NY1tU=";
   };
 
+  patches = lib.optionals stdenv.isDarwin [ ./threads.patch ] ++ [
+    (fetchpatch {
+      name = "missing-override.patch";
+      url =
+        "https://github.com/souffle-lang/souffle/commit/da2d778f0cca94f206686546fa56b9ffc738ad75.patch";
```

Comment:
```suggestion
      url = "https://github.com/souffle-lang/souffle/commit/da2d778f0cca94f206686546fa56b9ffc738ad75.patch";
```

Reply:
Done. It was nixfmt choice to put in a different line. I only kept the nixfmt suggestions for the additions since there were a lots of changes in this file if I go along with it.

Eitherway, lines merged. I'm not 100% the conventions here. It's my first PR to nixpkgs.

Thanks for the review.